src/Utils/GalleryUtils.ts

```diff
@@ -243,6 +243,11 @@ export function downloadItem(
         const notebook = JSON.parse(response.data) as Notebook;
         removeNotebookViewerLink(notebook, data.newCellId);
 
+        if (!data.isSample) {
+          const metadata = notebook.metadata as { [name: string]: unknown };
+          metadata.untrusted = true;
```

Comment:
>   metadata.untrusted = true; [](http://example.com/codeflow?start=7&length=29)

thoughts on this being on the server?

Reply:
We could also do this at the time of publishing server side. I don't see any down sides to that either.